Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -65,10 +69,20 @@ - (id)initWithDrawingAreaProxy:(WebKit::RemoteLayerTreeDrawingAreaProxy*)drawing
     if (self = [super init]) {
         _drawingAreaProxy = drawingAreaProxy;
         // Note that CADisplayLink retains its target (self), so a call to -invalidate is needed on teardown.
-        _displayLink = [CADisplayLink displayLinkWithTarget:self selector:@selector(displayLinkFired:)];
-        [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:NSRunLoopCommonModes];
-        _displayLink.paused = YES;
-        _displayLink.preferredFramesPerSecond = 60;
+        bool createDisplayLink = true;
+#if ENABLE(TIMER_DRIVEN_DISPLAY_REFRESH_FOR_TESTING)
+        NSInteger overrideRefreshRateValue = [NSUserDefaults.standardUserDefaults integerForKey:@"MainScreenRefreshRate"];
+        if (overrideRefreshRateValue) {
```

Comment:
This could be `if (NSInteger overrideRefreshRateValue = [NSUserDefaults.standardUserDefaults integerForKey:@"MainScreenRefreshRate"])` but I think its a little obfuscating